src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java

```diff
@@ -0,0 +1,72 @@
+package org.apache.accumulo.testing.performance.tests;
+
+import java.nio.charset.StandardCharsets;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.metadata.MetadataTable;
+import org.apache.accumulo.testing.performance.Environment;
+import org.apache.accumulo.testing.performance.PerformanceTest;
+import org.apache.accumulo.testing.performance.Report;
+import org.apache.accumulo.testing.performance.SystemConfiguration;
+import org.apache.hadoop.io.Text;
+
+public class HighSplitCreationPT implements PerformanceTest {
+
+  private static final int NUM_SPLITS = 10_000;
+  private static final int MIN_REQUIRED_SPLITS_PER_SECOND = 100;
+  private static final int ONE_SECOND = 1000;
+  private static final String TABLE_NAME = "highSplitCreation";
+  private static final String METADATA_TABLE_SPLITS = "123456789abcde";
+
+  @Override
+  public SystemConfiguration getSystemConfig() {
+    return new SystemConfiguration();
+  }
+
+  @Override
+  public Report runTest(final Environment env) throws Exception {
+    Report.Builder reportBuilder = Report.builder().id("high_split_creation")
+        .description("Evaluate the speed of creating many splits.")
+        .parameter("table_name", TABLE_NAME, "The name of the test table")
+        .parameter("num_splits", NUM_SPLITS, "The high number of splits to add.")
+        .parameter("min_required_splits_per_second", MIN_REQUIRED_SPLITS_PER_SECOND,
+            "The minimum average number of splits that must be created per second before performance is considered too slow.");
+
+    AccumuloClient client = env.getClient();
+    client.tableOperations().create(TABLE_NAME);
+    client.tableOperations().addSplits(MetadataTable.NAME, getMetadataTableSplits());
+
+    SortedSet<Text> splits = getTestTableSplits();
+
+    long start = System.currentTimeMillis();
+    client.tableOperations().addSplits(TABLE_NAME, splits);
+    long totalTime = System.currentTimeMillis() - start;
+    double splitsPerSecond = NUM_SPLITS / (totalTime / ONE_SECOND);
+
+    reportBuilder.parameter("splits_per_second", splitsPerSecond,
+        "The average number of splits created per second.");
+    reportBuilder.parameter("meets_performance_threshold",
+        splitsPerSecond > MIN_REQUIRED_SPLITS_PER_SECOND,
+        "Whether or not the average split creation meets or exceeds the minimum number of splits per second.");
```

Comment:
`reportBuilder.parameter` should be replaced with `reportBuilder.result` for the resulting value of the performance test. Results are written in a different section of the json file that gets written.

Reply:
The same should be done for the results of the other tests as well.

Reply:
@Manno15 thanks for the input, I'll update the code to do so.

Reply:
Changes made.